openedx/core/djangoapps/embargo/middleware.py

```diff
@@ -83,7 +83,7 @@ def process_request(self, request):
             if pattern.match(request.path) is not None:
                 return None
 
-        ip_address = get_ip(request)
+        ip_address, _ = get_client_ip(request)
```

Comment:
```suggestion
        ip_address = get_client_ip(request)[0]
```
Is there a reason this is inconsistent with the changes in other files?

Reply:
changed.